security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Restricting Fiddler
+description: "Using the Enterprise policy feature to disable specific Fiddler Everywhere features like remote connections, changing ports, cloud services, and others."
+slug: fe-restrict-policies
+publish: true
+position: 40
+---
+
+# Restricting Fiddler
+
+Users of Fiddler Everywhere often work in a secure corporate environment where specific policies are imposed. That means that administrators need to have options to limit or hardcode particular features of the Fiddler Everywhere application. With the Fiddler Everywhere Enterprise tier, administrators can control a set of functionalities through policy management systems on macOS and Windows.
+
+## Restricting Features on macOS
+
+
+Users on macOS with administrative privileges set policies through the [**defaults**](https://macos-defaults.com/) command and by using the `com.progress-telerik.fiddler` domain alongside the following policies:
+
+| Policy          | Description | Value Type |  Value to apply the policy
+|:-----------------|:----------------|:----------------|:-----------------
+| **DisableCloud**   | Disables sharing options  | integer  | 1
+| **DefaultProxyPort**   | Sets default proxy port  | integer  | A number between 0 and 95535
+| **DisableProxyPortChange**   | Disables the change of port  | integer  | 1
+| **DefaultBypassList**   | Sets default bypass list  | string  | Add URLs with “;” in between
+| **DisableBypassListChange**   | Disables the change of the bypass list | integer  | 1
+| **DisableAllowRemoteDevicesToConnect**   | Disables option to connect to remote devices | integer  | 1
+| **DisableAnalytics**   | Disables Fiddler's analytics  | integer  | 1
+
+Setting a policy through the **defaults** command
+```sh
+defaults write com.progress-telerik.fiddler DisableAllowRemoteDevicesToConnect 1
+```
+
+For the **DefaultProxyPort** policy, set the value to be integer as follows:
+```sh
+defaults write com.progress-telerik.fiddler DefaultProxyPort -integer 8899
+```
+
+Reading the set policies for the Fiddler Everywhere application
+```sh
+defaults read com.progress-telerik.fiddler
+```
+
+Deleting set policies for the Fiddler Everywhere application
+```sh
+defaults delete com.progress-telerik.fiddler
+```
+
+## Restricting Features on Windows
+
+Users on Windows with administrative privileges set policies through the Windows registry editor by setting the following policies:
```

Comment:
They should not edit the registries. IT teams set the configuration through other tools. The important information for them is:
- Where should they store the keys (path to the registries respected by Fiddler Everywhere)
- What are the keys, type of the values and example values.